### PR TITLE
Enocean Plugin improvements

### PR DESCRIPTION
**Pull request by aschwith**

### CODE REVIEW

**Comment by aschwith** [comment]
Hallo zusammen, ich habe keine Schreibrechte mehr für SmarthomeNG/plugins. Kann jemand mal den Pullrequest annehmen? Danke und beste Grüße

File: enocean/__init__.py [hunk 1]
```diff
@@ -172,6 +211,7 @@ def _process_packet_type_radio(self, data, optional):
             dBm = -optional[5]
             SecurityLevel = optional[6]
             logger.debug("enocean: radio message with additional info: subtelnum = {} / dest_id = {:08X} / signal = {}dBm / SecurityLevel = {}".format(subtelnum, dest_id, dBm, SecurityLevel))
+            self.logger.debug("enocean: radio message with additional info: subtelnum = {} / dest_id = {:08X} / signal = {}dBm / SecurityLevel = {}".format(subtelnum, dest_id, dBm, SecurityLevel))
 
         if sender_id in self._rx_items:
             logger.debug("enocean: Sender ID found in item list")
```
Comment (ohinckel) [inline enocean/__init__.py]: Seems here is also a `self.` missing in this logger cal.

File: enocean/__init__.py [hunk 2]
```diff
@@ -85,22 +86,47 @@
...
+SA_WR_CLIENTLEARNRQ    = 0x04          # Send Smart Ack Learn request (Client)
+SA_WR_RESET            = 0x05          # Send reset command to a Smart Ack client
+SA_RD_LEARNEDCLIENTS   = 0x06          # Get Smart Ack learned sensors / mailboxes
+SA_WR_RECLAIMS         = 0x07          # Set number of reclaim attempts
+SA_WR_POSTMASTER       = 0x08          # Activate/Deactivate Post master functionality
+
 SENT_RADIO_PACKET              = 0xFF
 SENT_ENCAPSULATED_RADIO_PACKET = 0xA6
 
 logger = logging.getLogger('EnOcean')
```
Comment (ohinckel) [inline enocean/__init__.py]: Do you still want to use this logger instance instead the one within the plugin instance?

File: enocean/__init__.py [hunk 3]
```diff
@@ -85,22 +86,47 @@
...
 
-class EnOcean():
 
+class EnOcean(SmartPlugin):
+    PLUGIN_VERSION = "1.1.2"
```
Comment (ohinckel) [inline enocean/__init__.py]: The plugin version should start with the version of SH. So I suggest to use `1.3.2` or `1.3.0` instead.

File: enocean/eep_parser.py [hunk 4]
```diff
@@ -105,14 +105,17 @@ def _parse_eep_A5_04_02(self, payload, status):
         result = {}
         result['ENG'] = 0.47 + (payload[0] * 1.5 / 66)       # voltage of energy buffer in Volts
         result['HUM'] = (payload[1] / 250.0 * 100)           # relative humidity in percent
-        result['TMP'] = -20.0 + (payload[2] / 250.0 * 40.0)  # temperature in degree Celsius from -20.0 degC- 60degC
+        result['TMP'] = -20.0 + (payload[2] / 250.0 * 80.0)  # temperature in degree Celsius from -20.0 degC - 60degC
         return result
 
     def _parse_eep_A5_08_01(self, payload, status):
         # Brightness and movement sensor, for example eltako FBH65TFB, RORG = 0x07
+        logger.debug("enocean: parsing A5_08_01: Movement sensor")
```
Comment (ohinckel) [inline enocean/eep_parser.py]: Don't want to create a logger instance in the parser object, too? Global variables are always a litte bit bad practice.

File: enocean/__init__.py (hunk 2 above)
Comment (aschwith) [inline enocean/__init__.py]: OK, I will remove the old logging completely.

File: enocean/__init__.py (hunk 3 above)
Comment (aschwith) [inline enocean/__init__.py]: Ok, done.

File: enocean/__init__.py (hunk 1 above)
Comment (aschwith) [inline enocean/__init__.py]: See, above

**Comment by aschwith** [comment]
Closing this one in favor of new pull request that includes above changes. Thanks for the review!

**Comment by ohinckel** [comment]
@aschwith thanks for taking care!

**Pull request closed without merging**